habitat_baselines/utils/common.py

```diff
@@ -74,6 +74,50 @@ def forward(self, x: Tensor) -> CustomFixedCategorical:
         return CustomFixedCategorical(logits=x)
 
 
+class CustomNormal(torch.distributions.normal.Normal):
+    def sample(
+        self, sample_shape: Size = torch.Size()  # noqa: B008
+    ) -> Tensor:
+        return super().rsample(sample_shape)
+
+    def log_probs(self, actions):
+        ret = (
+            super()
+            .log_prob(actions)
+            .view(actions.size(0), -1)
+            .sum(-1)
+            .unsqueeze(-1)
+        )
+        return ret
+
+
+class GaussianNet(nn.Module):
+    def __init__(
+        self,
+        num_inputs: int,
+        num_outputs: int,
+        std_min: float = 1e-6,
+        std_max: float = 1,
+    ) -> None:
+        super().__init__()
+
+        self.mu = nn.Linear(num_inputs, num_outputs)
+        self.std = nn.Linear(num_inputs, num_outputs)
+        self.std_min = std_min
+        self.std_max = std_max
+
+        nn.init.orthogonal_(self.mu.weight, gain=0.01)
+        nn.init.constant_(self.mu.bias, 0)
+        nn.init.orthogonal_(self.std.weight, gain=0.01)
+        nn.init.constant_(self.std.bias, 0)
+
+    def forward(self, x: Tensor) -> CustomNormal:
+        mu = torch.tanh(self.mu(x))
+        std = torch.clamp(self.std(x), min=self.std_min, max=self.std_max)
```

Comment:
Following up with the convo from slack, I do think we should have some configuration here since there isn't really an agreed upon way to predict STD.

Reply:
Added support for:

- Tanh activation
- no activation
- softplus activation
- std (min max provided by config)
- log_std (min max provided by config)